documentation/modules/cruise-control/con-optimization-proposals.adoc

```diff
@@ -94,10 +105,52 @@ m¦excludedBrokersForReplicaMove
 
 |===
 
+
+[discrete]
+=== Broker load
+
+The broker load is stored in a ConfigMap (with the same name as the KafakRebalance custom resource) as a JSON formatted string. This JSON string consists of a JSON object with keys for each broker IDs linking to a number of metrics for each broker
```

Comment:
You are missing a full stop at the end of this sentence.

Reply:
I have added it now.